src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,49 +1,75 @@
-import React from 'react';
-import { Loader } from '../Loader';
+import React, {useEffect, useState} from 'react';
+import {Loader} from '../Loader';
+import {User} from '../../types/User';
+import {Todo} from '../../types/Todo';
+import {getUser} from '../../api';
+
+type Props = {
+  todo: Todo
+  reset: () => void
+};
+
+export const TodoModal: React.FC<Props> = ({todo, reset}) => {
+  const [activeUser, setActiveUser] = useState<User>();
+
+  useEffect(() => {
+    getUser(todo.userId).then(user => setActiveUser(user));
+  }, [todo.id]);
 
-export const TodoModal: React.FC = () => {
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
 
-      {true ? (
-        <Loader />
-      ) : (
-        <div className="modal-card">
-          <header className="modal-card-head">
-            <div
-              className="modal-card-title has-text-weight-medium"
-              data-cy="modal-header"
-            >
-              Todo #2
-            </div>
+      {!activeUser
```

Comment:
If you have such a long conditions it is not readable to use ternary.
You can use  
```
{!activeUser && ...}
{activeUser && ...}
```